### PR TITLE
a couple spelling fixes

### PR DESCRIPTION
**Pull request by zoshkalin**
#### Summary

```SUMMARY: None```

fixes some spelling mistakes

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
There is a conflict in one file which should be resolved via command line. Otherwise looks good.

**Comment by Night-Pryanik** [comment]
That's definitely not a **couple** of fixes. Good work!

**Comment by Brian-Otten** [comment]
Solid work here.

File: data/json/npcs/refugee_center/surface_refugees/NPC_Dana_Nunez.json [hunk 1]
```diff
@@ -356,7 +356,7 @@
     "copy-from": "sourdough_starter",
     "looks_like": "sourdough_starter",
     "name": "Dana's family sourdough starter",
-    "description": "This crusty old jar has a glow-in-the-dark millenium falcon decal on the side, and a label that reads, \"Landough Calrisean.  Dana's.  Do not touch on pain of death\".  It's been abandoned for some time and doesn't look like it could be readily salvaged as a sourdough culture, but maybe an experienced sourdough baker could manage it."
+    "description": "This crusty old jar has a glow-in-the-dark millennium falcon decal on the side, and a label that reads, \"Landough Calrisean.  Dana's.  Do not touch on pain of death\".  It's been abandoned for some time and doesn't look like it could be readily salvaged as a sourdough culture, but maybe an experienced sourdough baker could manage it."
```
Comment (geophree) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Dana_Nunez.json]: `millennium falcon` -> `Millennium Falcon`

File: data/json/npcs/TALK_TRUE_FOODPERSON.json [hunk 2]
```diff
@@ -353,7 +353,7 @@
         "trial": { "type": "PERSUADE", "difficulty": 0, "mod": [ [ "TRUST", 2 ], [ "VALUE", 2 ] ] },
         "success": {
           "topic": "TALK_FOODPERSON_SIDEKICK",
-          "effect": { "npc_add_var": "npc_succes_team_team", "type": "dialogue", "context": "foodperson", "value": "yes" }
+          "effect": { "npc_add_var": "npc_success_team_team", "type": "dialogue", "context": "foodperson", "value": "yes" }
```
Comment (geophree) [inline data/json/npcs/TALK_TRUE_FOODPERSON.json]: You didn't cause this bug, but I'm pretty sure it should be `npc_success_team` (without the extra `_team`).

**Pull request merged**